### DISCUSSION TITLE
Authors aren't shown anymore

### DISCUSSION THREAD

**Discussion by Alseidon** [FreshRSS/FreshRSS#6095] [Q&A] [answered]
I didn't check my arXiv RSS feed for quite a while (at least a month) and when I checked it today, paper authors didn't show up anymore, even though I set them to. The only thing I found out changed was [this update](https://blog.arxiv.org/2024/01/31/attention-arxiv-users-re-implemented-rss/) where they say: "An author list is now provided for each paper". Is there a way I can fix this?

**Comment by Alkarex** [comment, marked as answer]
The feeds seems to use `dc:creator` with HTML inside to indicate the authors, which seems to require a patch on our side to be supported.
Would require a bit more investigation.
Example: https://arxiv.org/rss/astro-ph

**Reply by Alseidon** [reply]
I was wondering if this was the issue, as I'm not too familiar with RSS feeds. Thanks!

**Reply by keynet** [reply]
I have a feed that has no dc:creator tag, and as a result shows up on Mailchimp with author as "anonymous".
Would it be possible to add an override for missing tags, or certain tags, or perhaps in this case default it to the source name if missing?
Example: (it's UK Parliament feed)
https://www.theyworkforyou.com/search/rss/?s=hammersmith
Thanks

**Reply by Alkarex** [reply]
@keynet At the moment, for such custom things, you would have to write a little FreshRSS extension, in which you can override the author for entries matching this feed URL.
